Sources/SwiftSyntax/SourceLength.swift

```diff
@@ -11,64 +11,46 @@
 //===----------------------------------------------------------------------===//
 
 /// The length a syntax node spans in the source code. From any AbsolutePosition
-/// you reach a node's end location by either adding its UTF-8 length or by
-/// inserting `lines` newlines and then moving `columns` columns to the right.
-public struct SourceLength {
-  public let newlines: Int
-  public let columnsAtLastLine: Int
+/// you reach a node's end location by adding its UTF-8 length.
+public struct SourceLength: Comparable {
```

Comment:
Seems like the same issue with `AbsolutePosition` applies here, though the `init(of: String)` initializer is kinda nice.